test/expected_output/client.rb

```diff
@@ -15,12 +15,12 @@
 # limitations under the License.
 
 module Google
-module Cloud
-module Speech
-module V1
-class SpeechClient
-end
-end
-end
-end
```

Comment:
Can you revert this indentation? The scope of this test does not currently include the `ProtoCompilerRunner#format_file` step, so the correct expectation (currently) should be without indentation. I am getting a `rake test` failure on this branch:

```sh
  1) Failure:
ClientTest#test_client [/Users/quartzmo/code/google/codez/gapic-generator-ruby/test/templates/client_test.rb:35]:
--- expected
+++ actual
@@ -15,13 +15,13 @@
 # limitations under the License.
 
 module Google
-  module Cloud
-    module Speech
-      module V1
-        class SpeechClient
-        end
-      end
-    end
-  end
+module Cloud
+module Speech
+module V1
+class SpeechClient
+end
+end
+end
+end
 end
 "
```